packages/web/src/components/search-bar/ConnectedSearchBar.jsx

```diff
@@ -79,14 +79,22 @@ class ConnectedSearchBar extends Component {
 
   onSubmit = (value) => {
     // Encode everything besides tag searches
-    if (!value.startsWith('#')) {
+    const pathname = '/search'
+    if (value.startsWith('#')) {
+      // perform tag search
+      const pathname = `/search`
+      this.props.history.push({
+        hash: value.split('#')[1].replace(/\s+/g, ''),
```

Comment:
does this work for multiple hashes?

also seems like if I searched `#whatever something else` we'd end up submitting `{ hash: 'whateversomethingelse' }` which doesn't seem correct

Reply:
yeaa i don't think we have any way to search multiple hash tags, but maybe good to follow up with

and i thought stripping all whitespace would be favorable than to end up w something like this

<img width="678" alt="Screenshot 2024-02-07 at 10 54 03 AM" src="https://github.com/AudiusProject/audius-protocol/assets/6413636/bb007fd3-f388-46d8-804b-73c3e0bd98ea">

Reply:
as for the history.hash value, i think these functions handle that 

https://github.com/AudiusProject/audius-protocol/blob/3d97d1ae66409ad659aee428846813f34124e9a9/packages/web/src/pages/search-page/helpers.ts#L26

https://github.com/AudiusProject/audius-protocol/blob/3d97d1ae66409ad659aee428846813f34124e9a9/packages/web/src/pages/search-page/helpers.ts#L60